uvicorn.sh

```diff
@@ -1 +1,3 @@
+export ENV=dev
```

Comment:
I don't like this. The default is already dev. If you put this here, you can't run the prod-debug env, or the test env, with this script.

Reply:
so...just delete the line?